modular_skyrat/modules/customization/modules/language/yangyu.dm

```diff
@@ -1,6 +1,6 @@
 /datum/language/yangyu
-	name = "Spatial Sino-Tibetan"
-	desc = "A genetic relationship between Chinese, Tibetan, Burmese and other Human languages of similar characteristics was first proposed in the early 19th century and is extremely popular even in the space age. Originating from Asia, this group of tongues is the second most spoken by Human and Human-derived populations since the birth of Sol Common - and was a primary contender to be the Sol Federation's official language."
+	name = "Spacial Sino-Tibetan"
```

Comment:
use Orbital Sino-Tibetan it goes hard
tie it into east asian orbital habitats around mars or smth